app/controllers/tag_controller.rb

```diff
@@ -289,8 +289,6 @@ def create
                            uid: current_user.uid,
                            body: " @#{current_user.username} has marked #{tagname.split(':')[1]} as a co-author. ")
 
-        end
```

Comment:
Is this a stray deletion? Maybe re-adding the `end` will fix some of the tests?

Reply:
Sure, on it!!